webservices/common/models/filings.py

```diff
@@ -125,14 +125,14 @@ def document_description(self):
 
     @property
     def amended_by(self):
-        if not self.superceded and not self.most_recent and self.amendment and len(self.amendment.longest_chain) > 0:
+        if len(self.amendment.longest_chain) > 0:
             index = self.amendment.longest_chain.index(self.file_number)
             return self.amendment.longest_chain[index + 1]
         else:
             return self.superceded
     @property
     def is_amended(self):
-        if self.superceded is not None or not self.most_recent:
+        if not self.superceded or not self.most_recent:
```

Comment:
Is this correct? Maybe I'm just tired, but this change seems to have introduced a bug? And could the function simply be:

```python
def is_amended(self):
  return self.superceded or not self.most_recent
```